### DISCUSSION TITLE
(β版)ダークモード時の記事の各見出し一覧部分のスクロールバーの色について

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#433] [要望・アイデア・不具合]
細かい点で恐縮ですが、記事ページの右の方にある見出し一覧部分のスクロールバーが（Macとかだと気にならないかもしれませんが）Windows環境だとちょっと浮いて目立っているような気がしておりまして、もしかしたらその辺もダークモードに合わせて黒いスタイル設定とか追加されるとより良くなるかもしれません！

![image](https://user-images.githubusercontent.com/25178045/219254022-638dba2f-30fa-4a90-af21-cf585d891b41.png)

GitHubとかだとダークモードで黒くするとスクロールバーも黒くなっております！

![image](https://user-images.githubusercontent.com/25178045/219254419-10b1da3a-7488-4fbf-ae06-c1cb80dc2290.png)

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
ありがとうございます🙏参考にさせていただきます！

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
大変お待たせしました！
記事の目次に表示されるスクロールバーや、その他のスクロールバーもダークモード仕様になるよう修正いたしました！
ページ全体のスクロールバーについては、影響範囲が大きいため今後改めて検討します🙏

**Reply by simon-ritchie** [reply]
@tanaka-asami415 
ご対応ありがとうございます！
